sdk/core/Azure.Core/src/ClientCertificate.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Security.Cryptography.X509Certificates;
+using Azure.Core;
+
+namespace Azure
+{
+    /// <summary>
+    ///
```

Comment:
nit: add docs

Reply:
thanks - I need to go do that in a few places.